twake/frontend/src/app/features/drive/api-client/api-client.ts

```diff
@@ -86,4 +103,20 @@ export class DriveApiClient {
         appendPublicToken(true),
     );
   }
+
+  static async search(searchString: string, options?: BaseSearchOptions) {
+    const companyId = options?.company_id ? options.company_id : Workspace.currentGroupId;
+    const query = `/internal/services/documents/v1/companies/${companyId}/search`;
```

Comment:
Did you check that we can't access restricted results through search ? I think on the backend side we had to add some access check.